Openssl recipe crashes on x86 arch

p4a branch: stable
buildozer: 0.33
bootstrap: sdl2
kivy: 1.10.0

Error message i get:
```
arm_arch.h:46:6: error: #error "unsupported ARM architecture"
```

--- a/pythonforandroid/recipes/openssl/__init__.py
+++ b/pythonforandroid/recipes/openssl/__init__.py
@@ -36,6 +36,8 @@ class OpenSSLRecipe(Recipe):
             return 'android-armv7'
         if 'arm' in aname:
             return 'android'
+        if 'x86' in aname:
+            return 'android-x86'
         return 'linux-armv4'
 
     def build_arch(self, arch):

